static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss

```diff
@@ -101,105 +101,117 @@ Show more button:
 .button--primary {
```

Comment:
May I have @sndrs 's opinion on the matter? Background: I had to decouple the `:link` state from the `:hover/focus/active` states in the mixin into two mixins, because the original version was doing too much for me. And undoing styles is not really my thing.